UPGRADING.md

```diff
@@ -68,7 +68,7 @@ In `google-cloud-build<2.0.0`, parameters required by the API were positional pa
 
 In the 2.0.0 release, all methods have a single positional parameter `request`. Method docstrings indicate whether a parameter is required or optional.
 
-Some methods have additional keyword only parameters. The available parameters depend on the [`google.api.method_signature` annotation](https://github.com/googleapis/googleapis/blob/master/google/devtools/cloudbuild/v1/cloudbuild.proto#L82) specified by the API producer.
+Some methods have additional keyword only parameters. The available parameters depend on the parameters specified by the API producer.
```

Comment:
```suggestion
Some methods have additional keyword-only parameters. The available parameters depend on the [`google.api.method_signature` annotation](https://github.com/googleapis/googleapis/blob/master/google/devtools/cloudbuild/v1/cloudbuild.proto#L82) provided by the API producer.
```
The linked-to proto is not in this repo, so keeping the link is fine IMO.

 Alternately, if we want to remove the link, we could say `s/annotation specified/specification provided/` to avoid too much repetition of the word "parameter."